docs/quick-launch.md

```diff
@@ -0,0 +1,125 @@
+---
+id: quick-launch
+title: How to Launch RSPeer Automatically
+sidebar_label: How to Launch RSPeer Automatically
+---
+RSPeer's *Quick Launch* feature allows users to start the client with a
+preconfigured Runescape account, proxy, and script without any user interaction.
+Using this feature, users are able to spawn clients through the Bot Management
+interface, developers are able to quickly test their scripts without manually
+activating them, and large-scale deployments are able to run in a completely
+automated fashion.
+
+## Basic Usage
+
+The easiest way to generate a configuration is through the QuickLaunch
```

Comment:
I actually just found this feature in the last update. I'm not too sure how it's intended to be used yet. If someone has some insight on this, I can update this page with some more details.